docs/conf.py

```diff
@@ -0,0 +1,189 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# onlineweb4 documentation build configuration file, created by
+# sphinx-quickstart on Fri Dec  8 23:37:19 2017.
+#
+# This file is execfile()d with the current directory set to its
+# containing dir.
+#
+# Note that not all possible configuration values are present in this
+# autogenerated file.
+#
+# All configuration values have a default; values that are commented out
+# serve to show the default.
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import django
+import os
+import sys
+from django.conf import settings
+
+sys.path.insert(0, os.path.abspath('../'))
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "onlineweb4.settings")
+#settings.configure()
```

Comment:
Why is this commented out?

Reply:
Tried a bunch of stuff to make Sphinx works with Django and forgot to remove some of it.